bokeh/io/export.py

```diff
@@ -159,25 +162,12 @@ def export_svgs(obj, filename=None, height=None, width=None, webdriver=None, tim
 
     return filenames
 
-# this is part of the API for this module
-from .webdriver import terminate_webdriver ; terminate_webdriver
-from .webdriver import webdriver_control ; webdriver_control
```

Comment:
@mattpap These breaking changes should have gotten a note in the release migration notes.